cmd/dp-dimension-importer/main.go

```diff
@@ -138,9 +139,9 @@ func main() {
 }
 
 func newConsumer(kafkaAddr []string, topic string, namespace string) *kafka.ConsumerGroup {
-	consumer, err := kafka.NewConsumerGroup(kafkaAddr, topic, namespace, kafka.OffsetNewest)
+	consumer, err := kafka.NewSyncConsumer(kafkaAddr, topic, namespace, kafka.OffsetOldest)
```

Comment:
Should we be setting the kafka offset to `kafka.OffsetOldest`?

Reply:
yes